src/ExportMenu.php

```diff
@@ -1391,7 +1391,11 @@ public function generateRow($model, $key, $index)
             } else {
                 $value = '';
             }
-            $format = ArrayHelper::getValue($column->contentOptions, 'cellFormat', null);
+            $contentOptions = $column->contentOptions;
+            if (is_callable($contentOptions)) {
+                $contentOptions = $contentOptions($model, $key, $index);
```

Comment:
I'm not sure, but the closure seems to have the following signature:
`function ($model, $key, $index, $column)`

Is it possible to implement it?

Reply:
A reference: https://github.com/kartik-v/yii2-grid/blob/master/src/ActionColumn.php#L120

Reply:
@bscheshirwork  Thank you for patch but @awwar is right. It requires exactly 4 columns :D .

Reply:
Thanks! It is correct now?

Reply:
Yep, now it's valid! 
But let make condition more strict. Change it from `if (is_callable($contentOptions)) {` to `if ($contentOptions instanceof Closure) {`.

Reply:
how about something like `[classname, methodname]`? this is callable, but not \Closure

Reply:
Yeah, it's look like good solution and it will work, but!
In kartik docs - https://github.com/kartik-v/yii2-grid/blob/master/src/ActionColumn.php#L118 `contentOptions` can be `array` or `\Closure`.
In yii2 docs - https://github.com/yiisoft/yii2/blob/master/framework/grid/Column.php#L105 condition looks like this.
So, if u pass obj with  `__invoke()` magic method (https://www.php.net/manual/ru/language.oop5.magic.php#object.invoke) it's will work on export but do not work in view.

In short, let's focus on the documentation :)

But if you deem it necessary to leave it this way, ok, let's leave it this way.